pkg/security/securitycontextconstraints/byrestrictions.go

```diff
@@ -116,6 +114,9 @@ func volumePointValue(scc *securityapi.SecurityContextConstraints) points {
 		default:
 			hasNonTrivialVolume = true
 		}
+		if hasHostVolume {
+			break
```

Comment:
this is different than what the code used to do.

Reply:
the code was wrong I think, break inside case won't break the for and i think this assumed the for will break

Reply:
>the code was wrong I think,

No, it wasn't wrong but it was slower than it can be.

> break inside case won't break the for

yes, exactly

Reply:
> the code was wrong I think, break inside case won't break the for and i think this assumed the for will break

As long as that is what it is supposed to do.

Reply:
Don't worry, we have tests for that function :)